src/helm/benchmark/scenarios/scenario.py

```diff
@@ -49,39 +49,39 @@ def unpack_tag(tag: str) -> Tuple[str, str]:
     return key, value
 
 
-class Input(ABC):
+@dataclass(frozen=True)
+class Input:
```

Comment:
This should be `TextInput`, which derives from `Input`, right?

Reply:
I thought since we will still need `text` in vhelm, we would keep `text` in the base classes (`Input` and `Output`). Otherwise, we would have to cast `input` as `TextInput` whenever we need to access `text`.

Reply:
But there are inputs that are going to be image only or a sequence of text, image, etc.?  It seems like eventually we have to deal with this problem...